---
layout: archive
title: "Qin An"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

<iframe width="100%" height="800" src="/files/paper1.pdf">

![some discription](/files/paper1.pdf "some discription")

## About

Ph.D. with 6 years of training and working experience in biology, statistics and bioinformatics. Experienced in generating novel biological insights and actionable hypothesis, by analyzing large-scale, complex biological datasets using bioinformatic tools, statistical methods and machine learning algorithms. Demonstrated ability in innovating new computational pipelines and algorithms, to process, analyze, and interpret various types of NGS data, especially large-scale single-cell RNA-seq and single-cell multi-omics datasets. 

## Skills 

* Programming: R, Python, Shell script. 
* High-performance computational cluster and UNIX/Linux command line environment. 
* Machine learning: unsupervised clustering, dimensional reduction, linear regression, logistic regression. 
* Probability and statistics: descriptive statistical analysis, statistical distribution, hypothesis test. 
* Hands-on experience with data analysis of various types of NGS data, including single-cell RNA-seq (10X and SMART-seq2), single-cell ATAC seq, single-cell multimodal profiling techniques (CITEseq), bulk RNA-seq, WGS, WGBS, and WES.  
* Bioinformatics analysis pipeline, software, and databases. 
* Teamwork in collaborative environments in both pharmaceutical company and academia. 

## Education 

Sep 2015 – May 2020
:   **Ph.D. in Human Genetics**, University of California, Los Angeles                                                                         

Sep 2011 – Jul 2015
:   **B.S. in Life Science**, Wuhan University, China                                                                  

## Work experience 

Scientific Investigator
GlaxoSmithKline, Upper Providence, PA Jan 2021– present

Hired and manage two graduate school level student interns
 Evaluate, apply and develop statistical methods and computation tools for the analysis of sequencing data and cell images
• Lead the statistical analysis for multiple high-throughput screens
• Co-lead a cross-functional effort to build computational solutions for analyzing data from new technologies



Postdoctoral Research Fellow (Advisor: Joseph R. Ecker) 
Salk Institute, San Diego, CA                                                                                       Sep 2020 – Jan 2021  
 
 
Computational Biologist Internship 
GSK, Upper Providence, PA                                                                           Jun 2020 – Aug 2020 
•	Led the internal analysis of Accelerating Medicines Partnership (AMP) Phase II dataset in rheumatoid arthritis (RA). 
•	Established a workflow for integrative dimensional reduction and clustering of a large-scale single-cell dual-omics dataset (involving 85 patients and over 500k cells), generated using the CITE-seq protocol.  Performed analysis of additional data generated by bulk RNA seq, CEL-Seq2, and CyTOF approaches.  
•	Identified a patient subtype with a unique disease phenotype. Identified a rare cell type associated with the severity of the disease using robust statistical frameworks, which provides novel insights about the pathological mechanism of the disease. This cell type could be a potential target for treatment development. 
•	Addressed specific biological questions from internal research groups using the AMP dataset.  
•	Weekly report to the manager and final report to the department. 
 
 
Graduate Student Researcher (Advisor: Guoping Fan) 
UCLA, Los Angeles, CA                                                                                                        Sep 2015 – Jun 2020 
Single-cell multimodal sequencing: 
•	Studied how DNA methylation in sensory neurons regulates neuronal regeneration after peripheral nerve injury.  
o Developed computational pipeline for integrative analysis of single-cell transcriptome, single-cell DNA methylome and scMT-seq datasets (simultaneous profiling of transcriptome and DNA methylome from a single cell), including 1) integrative analysis of single-cell RNA-seq, single-cell DNA methylome, and single-cell multi-omics data; 2) integrative analysis of allelic DNA methylation and allelic gene expression; 3) analyzing how DNA methylation on transcription factor binding sites regulates neuronal regeneration after peripheral nerve injury. 
 
Single-cell RNA-sequencing: 
•	Revealed the transcriptome dynamics of human embryos during peri-implantation development by single-cell RNA seq (more than 500 samples profiled by SMART-seq2 protocol). 
o	Developed a novel algorithm that faithfully identified a novel transcription factor TBX3 regulating trophoblast differentiation. 
o	Identified genetic networks regulating peri-implantation trophoblast development. 
o	Determine when trophoblast differentiation happens in human embryos. 
 
•	Studied mechanism regulating early human retinogenesis using hESC derived retinal organoids and single-cell RNA seq. 
o	Deconstructed the temporal progression of retinal progenitor cells (RPC) during early human retinogenesis. Identified two distinctive subtypes of RPCs with unique molecular profiles. 
o	Dissected molecular dynamics underlying RPC commitment. 
o	Identified a novel gene (CCND1) promotes retinal neurogenesis. 
 
Droplet single-cell sequencing: 
•	Studying the impact of Dnmt1 point mutations on neurodegeneration using 10X genomics single-cell RNA-seq and single-cell ATAC-seq.  
 
Circulating cell-free DNA: 
•	Revealed the possibility of non-invasively monitoring cancer progression and tumor burden in human patients by measuring size of mitochondria-derived circulating cell free DNA fragments.  
•	Constructed sequencing libraries from tumor lesion samples and plasma cell free DNA, and performed formal data analysis of WGS, WES, and WGBS data.  
 
Collaboration projects: 
•	Performed bioinformatic analysis for additional 5+ ongoing projects, including 10X genomics single cell RNA-seq, bulk-RNA seq, and 5’end enriched single cell RNA-seq data (scRCAT-seq).  


